Python_Scripts/compareStateTrajectories_BioModels_2.py

```diff
@@ -3,50 +3,31 @@
 # => creates two important .tsv files
```

Comment:
This Docstring in the beginning of the file is little to not indicative at all. I got no clue, what this script will do...
Could we please change that?

Reply:
changed